pulp_2_tests/tests/rpm/api_v2/test_updateinfo.py

```diff
@@ -484,8 +484,8 @@ def test_02_unassociate_publish(self):
         Fetch ``updateinfo.xml``. Verify that an ``<update>`` with an ``<id>``
         of ``pulp_2_tests.constants.RPM_ERRATUM_ID`` is not present.
         """
-        client = api.Client(self.cfg, api.json_handler)
-        client.post(urljoin(self.repo['_href'], 'actions/unassociate/'), {
+        self.client = api.Client(self.cfg, api.json_handler)
```

Comment:
```suggestion
```

No need to redefine the `self.client` here, it is already created on SetUpClass

Reply:
@rochacbruno good call. Missed that. Thank you.